src/legacy/core_plugins/timelion/public/panels/timechart/timechart.js

```diff
@@ -19,10 +19,9 @@
 
 import Panel from '../panel';
 import { i18n } from '@kbn/i18n';
-import panelRegistry from '../../lib/panel_registry';
 
-panelRegistry.register(function timeChartProvider(Private) {
+export function timeChartProvider(Private) {
```

Comment:
Can we remove Private here?

Reply:
It would be great to know how to do that nicely

Reply:
@lukeelmers Maybe you can advise something? :)

Reply:
@Avinar-24  please open one of plugin which was already shimmed and navigate to 'shim' folder. Here you can find answer

Reply:
The plugin has been shimmed